dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -197,45 +197,45 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Imprimare"
 FONT 8, "MS Shell Dlg"
 {
- LTEXT "Imprimantă:", stc1, 6, 6, 40, 9
+ LTEXT "Imprimantă", stc1, 6, 6, 40, 9
  LTEXT "", stc2, 60, 6, 150, 9
- GROUPBOX "Volumul de imprimare", grp1, 6, 30, 160, 65, BS_GROUPBOX
+ GROUPBOX "Interval de imprimare", grp1, 6, 30, 160, 65, BS_GROUPBOX
  RADIOBUTTON "T&ot", rad1, 16, 45, 60, 12
- RADIOBUTTON "S&elecția", rad2, 16, 60, 60, 12
- RADIOBUTTON "&Paginile", rad3, 16, 75, 60, 12
- DEFPUSHBUTTON "I&mprimă", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
- PUSHBUTTON "A&nulează", IDCANCEL, 206, 24, 56, 14, WS_GROUP | WS_TABSTOP
- PUSHBUTTON "Opți&uni…", psh1, 206, 46, 56, 14, WS_GROUP | WS_TABSTOP
- LTEXT "&De la:", stc3, 60, 80, 30, 9
- LTEXT "&La:", stc4, 120, 80, 30, 9
- LTEXT "&Calitatea imprimării:", stc5, 6, 100, 76, 9
+ RADIOBUTTON "S&elecţia", rad2, 16, 60, 60, 12
+ RADIOBUTTON "&Pagini", rad3, 16, 75, 60, 12
+ DEFPUSHBUTTON "OK", IDOK, 206, 6, 56, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "Revocare", IDCANCEL, 206, 24, 56, 14, WS_GROUP | WS_TABSTOP
+ PUSHBUTTON "&Preferinţe", psh1, 206, 46, 56, 14, WS_GROUP | WS_TABSTOP
```

Comment:
You changed line 208 from "Propr&ietari" to "&Preferinţe" (as a reaction to a comment of mine). Maybe this was the wrong place though. I raised this question actually for line 361 and not for line 208.
Because now line 208 "&Preferinţe" does collide in the accelerator with `RADIOBUTTON "&Pagini", rad3, 16, 75, 60, 12`

Reply:
I didn't pay attention. I made a dumb mistake. I will revert this change.

Reply:
Fixed!

Reply:
Resolved.